src/null-safety/index.md

```diff
@@ -115,9 +113,9 @@ To resolve these issues, check for
 of any packages you installed from pub.dev, and [migrate](#migrate) all
 of your own source code to use sound null safety.
 
-As of late April 2023, **Dart 3 beta** is available from the Dart beta channel
-and the Flutter beta channel; see [the download page][] for details.
-We recommend you test your code for Dart 3 compatibility using that release:
+**Dart 3** has been released to the Dart and Flutter stable channels.
```

Comment:
nit: I'm not sure it makes sense to say "Dart 3" is released to the Flutter stable channel, since it's not released separately from Flutter. Perhaps "available" is still more accurate, at least in terms of Flutter.

Reply:
+1 for "Dart 3 is available in the Dart and Flutter stable channels"